client/src/pages/Details.js

```diff
@@ -88,7 +94,7 @@ function Details() {
           {filteredMenuItems.map((data) => {
             return (
               <div key={data.id}>
-                <img src={data.img} alt="images" />
+                <img src={data.img} alt="images" transform={getCloudinaryURL} />
```

Comment:
I don't think `transform` is a standard HTML image element property.  Did you mean to bring in a react `Img` component from a library somewhere?

Reply:
If not, I think you're SUPER close to the implementation.  You'll need to replace the `src` attribute with the `getCloudinaryURL` return value.  Let's discuss it today if there's time.